docker/frontend/Dockerfile

```diff
@@ -24,15 +29,19 @@ COPY --from=builder /build/nginx-mempool.conf /etc/nginx/conf.d/
 RUN chmod +x /patch/entrypoint.sh
 RUN chmod +x /patch/wait-for
 
-RUN chown -R 1000:1000 /patch && chmod -R 755 /patch && \
-        chown -R 1000:1000 /var/cache/nginx && \
-        chown -R 1000:1000 /var/log/nginx && \
-        chown -R 1000:1000 /etc/nginx/nginx.conf && \
-        chown -R 1000:1000 /etc/nginx/conf.d
+RUN chown -R $UID:$GID /patch && chmod -R 755 /patch && \
+        chown -R $UID:$GID /var/cache/nginx && \
+        chown -R $UID:$GID /var/log/nginx && \
+        chown -R $UID:$GID /etc/nginx/nginx.conf && \
+        chown -R $UID:$GID /etc/nginx/conf.d
 RUN touch /var/run/nginx.pid && \
-        chown -R 1000:1000 /var/run/nginx.pid
+        chown -R $UID:$GID /var/run/nginx.pid
+
+RUN deluser nginx \
+    && addgroup -g $GID -S nginx \
+    && adduser -S -D -H -u $UID -h /var/cache/nginx -s /sbin/nologin -G nginx -g nginx nginx
```

Comment:
Why is this needed?

Reply:
Alpine comes with no way that I know of, to modify the UID/GID of an existing user, besides directly changing config-files. 
In the backend-image I used `usermod`, which exists in Debian but not Alpine.
As I don't want to write directly to config-files, I simply delete the old user and create a new one with the correct IDs.